src/vivarium_gates_mncnh/data/loader.py

```diff
@@ -343,6 +343,8 @@ def load_lbwsg_rr(
     data = load_standard_data(key, location, years)
     data = data.query("year_start == 2021").droplevel(["affected_entity", "affected_measure"])
     data = data[~data.index.duplicated()]
+    cap_mask = data > 100.0
```

Comment:
This is the only code update. This means relative risks were updated and then hidden from this pr but PAFs were also calculated again using the new relative risks.